examples/fluids/qfunctions/stg_shur14.h

```diff
@@ -20,16 +20,14 @@
 #include <ceed.h>
 #include <stdlib.h>
 #include "stg_shur14_type.h"
+#include "utils.h"
 
 #ifndef M_PI
 #define M_PI    3.14159265358979323846
 #endif
```

Comment:
Should this go to the utils file as well?

Reply:
I had that thought, meant to bring it up. I think it makes sense for it to go into utils.h